applications/StructuralMechanicsApplication/tests/test_patch_test_truss.py

```diff
@@ -282,7 +253,7 @@ def _check_results_cable(self,mp,Force_X):
 
 
         self.assertAlmostEqual(disp_u_2, 0.022296019142446475,6)
-        self.assertAlmostEqual(r_u_1, -Force_X,6)
+        self.assertAlmostEqual(r_u_1, -100000000.00093779,6)
```

Comment:
shouldn't this be exactly 100000000

Reply:
yes, but you yourself asked for this change in #4286

Reply:
It is  a very small change...

Reply:
fixed properly